projects/cdk/pipes/emails/emails.options.ts

```diff
@@ -0,0 +1,22 @@
+import {tuiCreateToken} from '@taiga-ui/cdk/utils/miscellaneous';
+
+export const TUI_EMAIL_PIPE_OPTIONS = tuiCreateToken([
+    'me.com',
```

Comment:
nit: Would be great to sort them by popularity, pretty sure gmail should go first :)

Reply:
Also let's keep the list a bit shorter, only showing most popular options, so that there is no scroll in the dropdown. Lists UX best practice is to keep it below 7 items.